src/plugins/good-meetup-tracking.js

```diff
@@ -91,37 +72,29 @@ class GoodMeetupTracking extends Stream.Transform {
 	 * @return {Object} the output of calling the next transform in the chain
 	 */
 	_transform(event, enc, next) {
-		// log the data to stdout for Stackdriver
 		const eventData = JSON.parse(event.data);
+
+		// log readable tracking data - non-encoded
 		console.log(JSON.stringify({
 			message: 'Tracking log',
 			payload: eventData,
 		}));
 
-		const record = this._settings.schema.toBuffer(eventData);
+		const { schema, logData } = this._settings;
+
+		const record = schema.toBuffer(eventData);
 
 		const eventDate = new Date(parseInt(eventData.timestamp, 10));
-		const data = {
-			name: 'Activity',
+		const analytics = {
 			record: record.toString('base64'),
-			version: 3,
-			schemaUrl: 'gs://meetup-logs/avro_schemas/Activity_v3.avsc',
+			schemaUrl: `gs://meetup-logs/avro_schemas/${schemaDef.name}_${schemaDef.doc}.avsc`,
```

Comment:
👍 

This will come in handy when the schema potentially gets loaded from somewhere else